mephisto/core/task_launcher.py

```diff
@@ -188,6 +192,9 @@ def launch_units(self, url: str) -> None:
         )
         self.units_thread.start()
 
+    def get_assignment_thread_flag(self) -> bool:
```

Comment:
Nit: ideally class methods should be named closer to the intent that they are trying to match, because down the road it's not necessarily clear what the `assignment_thread_flag` is supposed to be representing. `get_assignments_are_all_created` for instance is more in-line with the intent for this flag.